content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,331 @@ Download the Istio release which includes installation files, samples, and the
     $ cd istio-{{< istio_full_version >}}
     {{< /text >}}
 
-    The installation directory contains:
+1.  Add the `istioctl` client to your path (Linux or macOS):
 
-    - Installation YAML files for Kubernetes in `install/kubernetes`
-    - Sample applications in `samples/`
-    - The [`istioctl`](/docs/reference/commands/istioctl) client binary in the `bin/` directory. `istioctl` is
-      used when manually injecting Envoy as a sidecar proxy.
+    {{< text bash >}}
+    $ export PATH=$PWD/bin:$PATH
+    {{< /text >}}
 
-1.  Add the `istioctl` client to your path, on a macOS or
-    Linux system:
+1.  For this tutorial, we use the `demo` profile. It's selected to have a good set
```

Comment:
This isn't a "tutorial", it's the setup instructions.

Reply:
fixed

Reply:
Also should we link to the configure profile page?
```suggestion
1.  For this tutorial, we use the `demo` [configuration profile](link). It's selected to have a good set
```

Reply:
fixed